@kiva/kv-components/vue/KvLightbox.vue

```diff
@@ -263,6 +264,8 @@ export default {
 
 		const onScreenClick = () => {
 			if (!preventClose.value) {
+				// This emit is only for MARS-199 experiment tracking and should be removed after results
```

Comment:
Should this be temporary or just left in? It seems like a reasonable addition to the component. If so, this comment can be removed.